index.html

```diff
@@ -1061,6 +1061,30 @@ <h3>Directory Service API</h3>
                     the Problem Details error responses may omit the `type` field which defaults
                     to "about:blank" and set `title` to the HTTP status text.
                 </p>
+
+                <p>
+                    The APIs set the HTTP status codes as defined in 
+                    <a href="https://datatracker.ietf.org/doc/html/rfc7231#section-6">Section 6</a> of [[RFC7231]].
+                    The list of used error codes include (but are not limited to) the following:
+                    <ul>
+                        <li>
+                            400 (Bad Request): Invalid client input in body, query, or headers.
+                            This is accompanied by an appropriate response message.
+                        </li>
+                        <li>
+                            401 (Unauthorized): The request lacks valid authentication credentials.
+                            This is often accompanied by a `WWW-Authenticate` header containing
+                            information on how the client can authenticate and repeat the request.
```

Comment:
I use "often accompanied by `WWW-Authenticate` header", because I'm not sure how many Restful APIs actually do that, how many non-browser clients support it, and if it is relevant for all security schemes.

Anyway, I think the details of this is beyond the scope of the current specification because we don't discuss how the security should be implemented. It doesn't make sense to go too deep in just one piece of the workflow.

Reply:
True, but we HAVE to do this for security bootstrapping.  Consolidation is good, and we care about security bootstrapping for WoT consumers specifically.